<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="user-PC" name="Test" tests="2" failures="1" timestamp="10 Oct 2016 06:15:30 GMT" time="0.075" errors="0">
  <testcase name="testFail" time="0.032" classname="pkgTest.NewTest">
    <failure type="java.lang.AssertionError" message="expected [This test case if failed.] but found [This test case is failed]">
      <![CDATA[java.lang.AssertionError: expected [This test case if failed.] but found [This test case is failed]
	at pkgTest.NewTest.testFail(NewTest.java:17)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
  <testcase name="testPass" time="0.005" classname="pkgTest.NewTest"/>
</testsuite> <!-- Test -->
